Rater.Business/Services/RatingService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Rater.API;
+using Rater.Business.Services.Interfaces;
+using Rater.Data.Repositories.Interfaces;
+using Rater.Domain.DataTransferObjects.RatingDto;
+using Rater.Domain.DataTransferObjects.UserDto;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rater.Business.Services
+{
+    public class RatingService : IRatingService
+    {
+
+
+        private readonly IRatingRepository _repo;
+        private readonly IUserService _userService;
+        private readonly IMapper _mapper;
+        public RatingService(IRatingRepository repo,IUserService userService,IMapper mapper)
+        {
+            _repo = repo;
+            _userService = userService;
+            _mapper = mapper;
+        }
+
+
+        public async Task<List<RatingForMetricResponseDto>> AddRatings(RatingRequestDto request)
+        {
+
+            if(request.RatingDetails.Count == 0 || request.RatingDetails == null)
+            {
+                throw new Exception("Bad Request of Ratings");                                       // If rating request is empty throw exception
```

Comment:
Remove the unnecessary comment and I believe there are some other more proper ways to validate request body F2